src/httpEnvelope.js

```diff
@@ -0,0 +1,39 @@
+var _ = require( "lodash" );
+
+function HttpEnvelope( req, res ) {
+	this.transport = "http";
+	this.context = req.context;
+	this.cookies = req.cookies;
+	this.data = req.body || {};
```

Comment:
Should this be cloned or are we okay with modifying the `req.body` by merging the `req.params` and `req.query` into it?